dashboard/src/components/Catalog/CatalogItem.tsx

```diff
@@ -11,29 +11,44 @@ interface ICatalogItemProps {
   chart: IChart;
 }
 
-class CatalogItem extends React.Component<ICatalogItemProps> {
-  public render() {
-    const { chart } = this.props;
-    const { icon, name, repo } = chart.attributes;
-    const iconSrc = icon ? `/api/chartsvc/${icon}` : placeholder;
-    const latestAppVersion = chart.relationships.latestChartVersion.data.app_version;
-    const repoTag = (
-      <Link className="ListItem__content__info_tag_link" to={`/catalog/${repo.name}`}>
-        {repo.name}
-      </Link>
-    );
-    return (
-      <InfoCard
-        key={`${repo}/${name}`}
-        title={name}
-        link={`/charts/${chart.id}`}
-        info={latestAppVersion || "-"}
-        icon={iconSrc}
-        tag1Content={repoTag}
-        tag1Class={repo.name}
-      />
-    );
+// 3 lines description max
+const MAX_DESC_LENGTH = 90;
+
+function trimDescription(desc: string): string {
```

Comment:
I do not think we need this function. We can rely on CSS instead. https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
Nope, as explained in the description, the ellipsis only work if it's just one line. I wanted to show at least three lines.